src/App.js

```diff
@@ -1,27 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
### OPTIONAL

- [ ]  It would be better for your App.js to be a functional component, functional components are the modern ways to write components 👍🏿